cluster.yml

```diff
@@ -4,11 +4,13 @@
     - { role: adduser, tags: adduser }
     - { role: download, tags: download }
     - { role: kubernetes/preinstall, tags: preinstall }
+    - { role: etcd, tags: etcd }
     - { role: docker, tags: docker }
+#   Available network plugins : calico, flannel. Choose only one of them
+    - { role: network_plugin/calico, tags: calico }
```

Comment:
just add the role network_plugin with a conditional depedency
` - { role: network_plugin }`

and in network_plugin/meta/mail.yml:
```
depedencies:
 - role: network_plugin/calico
   when: network_plugin == 'calico'
 - role: network_plugin/flannel
   when: network_plugin == 'flannel'
```